### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by AdrianBor**

### CODE REVIEW

**Comment by AdrianBor** [comment]
[Profile Page](https://trello.com/c/pQYLeHS2/113-profile-page)

**Comment by AdrianBor** [comment]
Friend section have been added BUT will be removed/post pone based on the difficulty to make it happend.

**Comment by AdrianBor** [comment]
Hi @lukaswelte , You can check my branch. You can hide the friend part. 
The first ME, I & Munich is when there is nothing so remove all the rest except contact.
When we start saving marks AND we are in the city, the trip list appear.

Just let me know if it'S not clear to telling you in a better way :)

**Comment by lukaswelte** [comment]
So this is ready to merge from your side now? If so please remove the WIP in the title.

**Comment by AdrianBor** [comment]
Yes for me it's done for the moment.

File: src/components/MarkMapMarker.css [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+  height: 14px;
   margin-left: -7px;
   margin-top: -7px;
-  border-radius: 30px;
+  border-radius: 7px;
   cursor: pointer;
-
-  background: url(../img/dots.png) no-repeat center center;
+  background: linear-gradient(45deg, rgba(251, 117, 127, 1), rgba(254, 86, 174, 1));
+  -webkit-box-shadow: 0px 0px 0px 4px rgba(255,255,255, 0.3);
```
Comment (lukaswelte) [inline src/components/MarkMapMarker.css]: you can remove the prefixed versions so -webkit -moz etc. as they are automatically added (less code :)

File: src/pages/ProfilDetail.css [hunk 2]
```diff
@@ -127,32 +112,41 @@
...
+  background-position: center center;
+  z-index: 9;
+}
+
+ul.ProfilDetail-friends li .ProfilDetail-friends-name {
+  margin-top: 21vw;
+}
+
+.ProfilDetail-poked {
+  -webkit-box-shadow: 0px 0px 0px 4px rgba(251, 117, 127, 0.8);
```
Comment (lukaswelte) [inline src/pages/ProfilDetail.css]: same here

**Review by lukaswelte — CHANGES_REQUESTED** [review]

**Comment by AdrianBor** [comment]
@lukaswelte Modifications done :)

**Review by lukaswelte — APPROVED** [review]

**Pull request merged**